app/_kong_plugins/ai-semantic-prompt-guard/examples/allow-and-deny-prompts.yaml

```
@@ -0,0 +1,45 @@
+description: 'Allows messages about a topic as long as they don't contain information about another topic. For example, only allow questions about cats that aren't related to dogs. Topics on the deny list take precedence over the allowed topics.'
```

```suggestion
description: |
  Allows messages about a topic as long as they don't contain information about another topic. 
  For example, only allow questions about cats that aren't related to dogs. Topics on the deny list take precedence over the allowed topics.
```
It doesn't like the `'` character in the description. This way it will ignore it and treat it like text.

---

That fixed it! I've applied these changes to all the plugin examples here.